### PR TITLE
Lint, style, CI, README improvements.

### PR DESCRIPTION
**Pull request by dmitshur**
Please see individual commit messages (expanded versions) for full details.

### CODE REVIEW

**Comment by dominikh** [comment]
@shurcooL is making me say "LGTM".

File: README.md [hunk 1]
```diff
@@ -1,7 +1,19 @@
-snippet-store
-=============
+# snippet-store [![Build Status](https://travis-ci.org/gopherjs/snippet-store.svg?branch=master)](https://travis-ci.org/gopherjs/snippet-store) [![GoDoc](https://godoc.org/github.com/gopherjs/snippet-store?status.svg)](https://godoc.org/github.com/gopherjs/snippet-store)
 
-Command snippet-store is a server for storing GopherJS Playground snippets.
+snippet-store is a server for storing GopherJS Playground snippets.
+
+It uses the same mapping scheme as the Go Playground, and when a snippet isn't found locally,
+it defers to fetching it from the Go Playground. This effectively augments our world of available
+snippets with that the Go Playground.
```
Comment (dominikh) [inline README.md]: `with that of`

Comment (dmitshur) [inline README.md]: Thanks for catching, fixed in 3f9e76b.

**Comment by dmitshur** [comment]
PTAL.

**Comment by neelance** [comment]
LGTM, except that I would prefer to use CircleCI instead of Travis. GopherJS itself is already using CircleCI and it is also free for open source projects.

**Comment by dmitshur** [comment]
Ok. We can discuss which CI system to use in the future. Let's start with Travis, since I already have a capable setup for it, and if we decide it's worth to switch to Circle, that can be another PR (where we can make sure all of the current functionality is preserved, etc.).

**Pull request merged**